README.md

```diff
@@ -2,68 +2,90 @@
+  * For each Floodlight device there be a Select entity to set the behaviour of the built-in motion sensor.
+* Number
+  * For each camera supporting WDR, a number entity will be setup to set the active value.
+  * For each camera a number entity will be created from where you can set the microphone sensitivity level.
+  * For each camera supporting Optical Zoom, a number entity will be setup to set the zoom position.
 
 It supports both regular Ubiquiti Cameras and the UniFi Doorbell. Camera feeds, Motion Sensors, Doorbell Sensors, Motion Setting Sensors and Switches will be created automativally for each Camera found, once the Integration has been configured.
 
 ## Table of Contents
 
-1. [Hardware Support](#hardware-support)
-2. [Prerequisites](#prerequisites)
-3. [Installation](#installation)
-4. [UniFi Protect Services](#special-unifi-protect-services)
-5. [UniFi Protect Events](#unifi-protect-events)
-6. [Automating Services](#automating-services)
+1. [UniFi Protect Support](#unifi-protect-support)
+2. [Hardware Support](#hardware-support)
+3. [Prerequisites](#prerequisites)
+4. [Installation](#installation)
+5. [UniFi Protect Services](#special-unifi-protect-services)
+6. [UniFi Protect Events](#unifi-protect-events)
+7. [Automating Services](#automating-services)
     * [Send a notfication when the doorbell is pressed](#send-a-notfication-when-the-doorbell-is-pressed)
-    * [Setting Recording or IR Mode](#automate-setting-recording-or-ir-mode)
     * [Person Detection](#automate-person-detection)
-    * [Input Slider for Microphone Sensitivity](#create-input-slider-for-microphone-sensitivity)
-7. [Enable Debug Logging](#enable-debug-logging)
-8. [Contribute to Development](#contribute-to-the-project-and-developing-with-a-devcontainer)
+    * [Input Slider for Doorbell Chime Duration](#create-input-slider-for-doorbell-chime-duration)
+8. [Enable Debug Logging](#enable-debug-logging)
+9. [Contribute to Development](#contribute-to-the-project-and-developing-with-a-devcontainer)
+
+## UniFi Protect Support
+
+In general, stable/beta version of this integration mirror stable/beta versions of UniFi Protect. That means:
+
+**Stable versions of this integration require the latest stable version of UniFi Protect to run.**
+
+**Beta versions / `master` branch of this integration require the latest beta version of UniFi Protect to run (or the latest stable if there is no beta)**
+
+We try our best to avoid breaking changes so you may need to use older versions of UniFi Protect with newer versions of the integration. Just keep in mind, we may not be able to support you if you do.
+
+## Docs for Old Versions
+
+If you are not using the latest beta of the integration, you can view old versions of this README at any time in Github at `https://github.com/briis/unifiprotect/tree/{VERSION}`. Example, docs for v0.9.1 can be found at [https://github.com/briis/unifiprotect/tree/v0.9.1](https://github.com/briis/unifiprotect/tree/v0.9.1)
```

Comment:
Merged the updates. Also added a second about access docs for older versions.

Reply:
Very good. Are you done so I can merge?

Reply:
Yep, go for it.